Samples/petstore/Python/swaggerpetstore/swagger_petstore.py

```diff
@@ -100,8 +99,10 @@ def add_pet_using_byte_array(
 
     def add_pet(
             self, body=None, custom_headers=None, raw=False, **operation_config):
-        """
-        Add a new pet to the store
+        """Add a new pet to the store
```

Comment:
Ok, found the "description + summary" scenario. Miss the ending "." for the summary.